app/helpers/application_helper.rb

```diff
@@ -51,4 +51,21 @@ def image_uploader_hint(uploader_field)
       number_to_human_size(uploader_field.size_range.max)
     )
   end
+
+  # ported from https://github.com/comfy/active_link_to/blob/master/lib/active_link_to/active_link_to.rb
+  def active_link_to(title, url, data: nil)
```

Comment:
I like this - it's such a small amount of code better to have it here where we can keep an eye on it.